updates to queries in ts (#1634)
temporalio/documentation: docs-src/typescript/how-to-handle-a-query-in-a-workflow-in-typescript.md

  - developer-guide
---

> Use [`handleQuery`](https://typescript.temporal.io/api/interfaces/workflow.workflowinboundcallsinterceptor/#handlequery) to return values inside a Workflow.

Reviewer: ```suggestion
Use [`handleQuery`](https://typescript.temporal.io/api/interfaces/workflow.workflowinboundcallsinterceptor/#handlequery) to handle Queries inside a Workflow.
```

> You make a Query with `handle.query(query, ...args)`.
> A Query needs a return value, but it can also take arguments.

Reviewer: ```suggestion
```

covered in other sections (sending & defining)

  - developer-guide
---

Use [`handleQuery`](https://typescript.temporal.io/api/interfaces/workflow.workflowinboundcallsinterceptor/#handlequery) to return values inside a Workflow.

> You make a Query with `handle.query(query, ...args)`. A Query needs a return value, but can also take arguments.

Reviewer 2: ```suggestion
You make a Query with `handle.query(query, ...args)`.
A Query needs a return value, but it can also take arguments.
```
